README.md

```diff
@@ -93,21 +93,21 @@ With the `updateId`, you can check the status (`enqueued`, `processing`, `proces
 
 ``` python
 # MeiliSearch is typo-tolerant:
-index.search('harry pottre')
+index.search('Mad Mxa')
 ```
 
 Output:
 
 ```python
 {
   "hits" => [{
-    "book_id" => 4,
-    "title" => "Harry Potter and the Half-Blood Prince"
+    "movie_id" => 4,
+    "title" => "Mad Max: Fury Road"
   }],
   "offset" => 0,
   "limit" => 20,
   "processingTimeMs" => 1,
-  "query" => "harry pottre"
+  "query" => "Mad Mxa"
 }
 ```
```

Comment:
This example didn't have a good format sorry.
```
```json
{
    "hits": [
        {
            "movie_id": 4,
            "title": "Mad Max: Fury Road,
            "genre": ["Adventure", "Science Fiction"]
        }
    ],
    "offset": 0,
    "limit": 20,
    "processingTimeMs": 1,
    "query": "Mad Mxa"
}
```
```